NitroxPatcher/Patches/Dynamic/LiveMixin_AddHealth_Patch.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Harmony;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.GameLogic.Helper;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.Core;
+using NitroxModel.DataStructures;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using NitroxModel_Subnautica.DataStructures;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic
+{
+    class LiveMixin_AddHealth_Patch : NitroxPatch, IDynamicPatch
```

Comment:
Let's see if we can refactor into its own class.  I think if you do this, then it would allow us to remove the need for simulation override (because the class can manage this stateful suppression).

Reply:
Consolidated the code into the LiveMixinManager and removed the simulationOverride.